Tests/OpenAPIRuntimeTests/Conversion/Test_Converter+Common.swift

```diff
@@ -54,6 +55,153 @@ final class Test_CommonConverterExtensions: Test_Runtime {
         }
     }
 
+    func testBestContentType() throws {
+        func testCase(
+            received: String?,
+            options: [String],
+            expected expectedChoice: String,
+            file: StaticString = #file,
+            line: UInt = #line
+        ) throws {
+            let choice = try converter.bestContentType(
+                received: received.map { .init($0)! },
+                options: options
+            )
+            XCTAssertEqual(choice, expectedChoice, file: file, line: line)
+        }
+
+        try testCase(
+            received: nil,
+            options: [
+                "application/json",
+                "*/*",
+            ],
+            expected: "application/json"
+        )
+        try testCase(
+            received: "*/*",
+            options: [
+                "application/json",
+                "*/*",
+            ],
+            expected: "application/json"
+        )
+        try testCase(
+            received: "application/*",
+            options: [
+                "application/json",
+                "*/*",
+            ],
+            expected: "application/json"
+        )
+        XCTAssertThrowsError(
+            try testCase(
+                received: "application/json",
+                options: [
+                    "whoops"
+                ],
+                expected: "application/json"
```

Comment:
Small nit but maybe pass an empty string/nil when we expect errors to be thrown instead, for clarity?

Reply:
Good idea, updated.